### PR TITLE
Kr product type

### PR DESCRIPTION
**Pull request by KaylaReid**
## Link to Ticket
Closes # [Link](https://github.com/NSS-Likeable-Lemurs/BangazonAPI/compare/kr-product-type?expand=1)

## Description of Proposed Changes
- Created model for ProductTypes
- Created controller for ProductTypes 
This controller allows the get all product types, get a single product type, post a new product type, put(edit) an existing product type, and delete a product type

## Steps to Test

Outline the steps to test

```
git fetch --all
git checkout <kr-product-type>
Add filler productType data into your SQL the seed data need to fill local database can be found in the SQL folder under the Bangazon.sql 
Test POST, PUT, DELETE, and GET in POSTMAN
```

## Impacted Areas in Application

List general components of the application that this PR will affect:

* productType Model and productType Controller

## Mentions @username

@askingalot 
@Akeen11 
@gretchenmarie 

## Definition of Done

1. The project must be fully documented. This includes the following:
    1. Complete README that documents the steps to install the code, how to install any dependencies, or system configuration needed.
    1. Every class must be documented with purpose, author, and methods.
    1. Every method must be documented with purpose and argument list - which itself must contain a short purpose for each argument.
1. The project must be able to run fully, and without errors, on each teammate's system.
1. Fulfills every requirement.
Every line of code has been peer reviewed.
1. For projects that require unit testing, core functionality must be identified and have at least one test for each.

### CODE REVIEW

**Review by Akeen11 — APPROVED** [review]
👍

**Review by gretchenmarie — APPROVED** [review]
good job :)

**Review by askingalot — COMMENTED** [review]
The link in the PR description doesn't go to the issue.

Please uncomment the SQL file so that I can rerun it and have it drop and recreate your database.

File: Bangazon/Controllers/ProductType.cs [hunk 1]
```diff
@@ -0,0 +1,158 @@
+﻿using System;
```
Comment (askingalot) [inline Bangazon/Controllers/ProductType.cs]: Rename this file to ProductTypeController

**Review by askingalot — COMMENTED** [review]
Remove the values controller.

File: Bangazon/appsettings.Development.json [hunk 2]
```diff
@@ -1,9 +1,11 @@
 {
+  "ConnectionStrings": {
```
Comment (askingalot) [inline Bangazon/appsettings.Development.json]: Remove this section form the appsettings.development.json file

File: Bangazon/Controllers/ProductType.cs (hunk 1 above)
Comment (KaylaReid) [reply Bangazon/Controllers/ProductType.cs]: Everything is ready to go!

**Review by askingalot — APPROVED** [review]

**Pull request merged**